torch/csrc/api/src/nn/init.cpp

```diff
@@ -146,6 +158,43 @@ Tensor uniform_(Tensor tensor, double low, double high) {
   return tensor.uniform_(low, high);
 }
 
+Tensor kaiming_uniform_(
+    Tensor tensor,
+    double a,
+    FanMode mode,
+    Nonlinearity nonlinearity) {
+  NoGradGuard guard;
+  Fan fan(tensor);
+  const auto gain = calculate_gain(nonlinearity, a);
```

Comment:
The first 9 lines of `kaiming_uniform_` and `kaiming_normal_` are duplicated. It'd be nicer if we put them in a function with a nice name in the anonymous namespace above and called it from both functions.